xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -34,13 +34,25 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+  <dl>
+    <dt>
+      <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
+    </dt>
+    <dd>
+      <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" 
+        class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    </dd>
+  </dl>
 </div>
 #end
 
 #if($editor == 'wiki')
-<div id="contentMeta">
-  <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
+<div id="contentMeta" class="ContentMetaLabel">
+  <dl>
```

Comment:
You could replace the ``<div id="contentMeta">`` with ``<dl id="contentMeta">`` in order to keep the HTML lighter.

Reply:
Ok I see.